components/MechPage/MechAgentMetrics.jsx

```diff
@@ -0,0 +1,26 @@
+import { SECTION_BOX_CLASS } from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Card } from 'components/ui/card';
+import Image from 'next/image';
+
+const CARD_BG =
+  'border-1.5 border-gray-200 rounded-2xl p-6 bg-gradient-to-t from-[#F2F4F7] to-white';
+
+export const MechAgentMetrics = () => (
+  <SectionWrapper customClasses={`${SECTION_BOX_CLASS} lg:py-14`}>
+    <div className="max-w-[750px] mx-auto text-center">
+      <Card className={`${CARD_BG} max-h-[250px] p-16`}>
+        <Image
+          alt="Mech metrics"
+          src="/images/optimus-page/optimus-metrics.svg"
```

Comment:
this is broken on preview
![image](https://github.com/user-attachments/assets/d224dcfa-fc34-4559-8089-e30b26ac7145)

Reply:
@atepem maybe if it's used on both pages, can name the icon just "metrics" and place in `/images`